README.md

```diff
@@ -84,39 +84,50 @@ transports may produce a high memory usage issue.
 
 ## Table of contents
 
-* [Motivation](#motivation)
-* [Quick Start](#quick-start)
-* [Usage](#usage)
-* [Table of Contents](#table-of-contents)
-* [Logging](#logging)
-  * [Creating your logger](#creating-your-own-logger)
-  * [Streams, `objectMode`, and `info` objects](#streams-objectmode-and-info-objects)
-* [Formats]
-  * [Combining formats](#combining-formats)
-  * [String interpolation](#string-interpolation)
-  * [Filtering `info` Objects](#filtering-info-objects)
-  * [Creating custom formats](#creating-custom-formats)
-* [Logging levels]
-  * [Using logging levels](#using-logging-levels)
-  * [Using custom logging levels](#using-custom-logging-levels)
-* [Transports]
+* [winston](#winston)
```

Comment:
The TOC in VSCode add-in auto-applied some consistency updates to the TOC, making indentation consistent (space vs tabs), and updating titles match the actual titles later in the document. Let me know if you'd prefer a PR without these touch-ups.

Reply:
Yeah given below comment let's maybe split this out / leave out this change for now

Reply:
Reverted